### ISSUE TITLE
Change color of visited stories

### ISSUE THREAD

**Issue by piotrromanowski** [kennygrant/gohackernews#21]
Change with something like 
`
.story a:visited {
    color: #551A8B;
}
`

**Comment by piotrromanowski** [comment]
Would've been nice if you had thrown this in the latest changes. I was going to leave a comment on https://golangnews.com/stories/2081-recent-changes-to-golang-news-meta but the login is broken. Or sign up is broken, not sure since the response to the request is incorrect.

**Comment by kennygrant** [comment]
Thinking of putting this in, sorry I was preoccupied with larger changes as lots changed under the hood. 

Re login what response do you see?

**Comment by kennygrant** [comment]
I like your idea of changing link colours - now that the larger changes are out of the way I've pushed up a colour change (used a grey instead).

Thanks for pointing this out I think I found the problem you were talking about - when creating a new account it was impossible to log in as the role wasn't being assigned (this happened because of recent changes to protect against injection of parameters, the role was being stripped). Let me know if you still see problems logging in now (perhaps open another issue).
